app/code/Magento/Inventory/Test/Mftf/Test/AdminAddConfigurableProductWithTextSwatchAttributeAndDefaultSourceTest.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="../../../../../../vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Test/etc/testSchema.xsd">
+    <test name="AdminAddConfigurableProductWithTextSwatchAttributeAndDefaultSourceTest">
```

Comment:
Please move this test to `Magento_InventoryConfigurableProduct` module

Reply:
I did put the test there because that is the home of AdminAddConfigurableProductWithDropDownAttributeAndCustomSourceTest.xml and AdminAddConfigurableProductWithDropDownAttributeAndDefaultSourceTest.xml which are pretty similar. Should I move it anyway?